Resources/Audio/Effects/Footsteps/attributions.yml

```diff
@@ -32,3 +32,11 @@
   license: "CC-BY-SA-3.0"
   copyright: "Taken and modified from tgstation (clownstep 1 and 2) by brainfood1183 (github)"
   source: "https://github.com/tgstation/tgstation/tree/f8ee37afc00bce1ad421615eaa0e4cbddd5eea90/sound/effects"
+
+- files:
+  - snake1.ogg
+  - snake2.ogg
+  - snake3.ogg
+  license: "CC-BY-SA-3.0"
+  copyright: "Provide a link to the source. Taken from https://zvukipro.com/"
```

Comment:
```suggestion
  copyright: "Taken from https://zvukipro.com/"
```

Reply:
done. I can give you the name of the author of the sounds if you need to.